src/main/java/org/mdbenefits/app/cli/TransmissionCommands.java

```diff
@@ -273,22 +302,48 @@ private void handleError(Transmission transmission, String errorKey, String erro
         updateTransmissionStatus(transmission, TransmissionStatus.FAILED, errorMap, false);
     }
 
+    /**
+     * This will handle the re-queuing of a transmission.  It will 1) log info about it 2) mark the transmission as QUEUED and 3)
+     * update the transmission status in the database.
+     *
+     * @param transmission the transmission to update
+     * @param messageKey   the message key to use when recording the message in the error map
+     * @param message      the message to put in the log or error map
+     * @param errorMap     the map of errors (messages) to get stored with the transmission in the db
+     */
+    private void handleRequeue(Transmission transmission, String messageKey, String message, Map<String, String> errorMap) {
+        log.warn("[Transmission {}]: {}", transmission.getId(), message);
+        if (messageKey != null) {
+            errorMap.put(messageKey, message);
+        }
+        updateTransmissionStatus(transmission, TransmissionStatus.QUEUED, errorMap, false);
+    }
+
     /**
      * Updates the transmission's status information, including the overall status, error messages and mark it sent (if
      * requested).
      *
      * @param transmission the transmission to update
      * @param status       the TransmissionStatus status
      * @param errorMap     a Map<String,String>
-     * @param markSent
+     * @param markSent     whether this should mark the record as sent to google drive or not
      */
     private void updateTransmissionStatus(Transmission transmission, TransmissionStatus status, Map<String, String> errorMap,
             boolean markSent) {
-        transmission.setStatus(status.name());
         transmission.setErrors(errorMap);
         if (markSent) {
             transmission.setSentAt(OffsetDateTime.now());
         }
+
+        // don't increment retry when just marking as in process
+        if (!status.equals(TransmissionStatus.TRANSMITTING)) {
+            int retryCount = transmission.getRetryCount();
+            retryCount++;
+            transmission.setRetryCount(retryCount);
```

Comment:
Question: if the retry count is == to the max, should we set status differently, or keep it as QUEUED? It will not get picked up again, but maybe the status should reflect this as well?

Reply:
How should we notify ourselves if an transmission fails to send after 5 tries?  We should put in some code here that causes us to be alerted.  Maybe an ERROR with a specific message and then a Datadog monitor for that message???   @stabai @spokenbird @analoo?

Reply:
It's easiest if there's a logging level that always triggers an alert. Do we have something like `CRITICAL` that I can set up to do that?

Reply:
Hmmm... no CRITICAL that I can see.